<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialServer</class>
 <widget class="QWidget" name="serialServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serialServer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#serialServer{background-color:transparent;border:0px;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
   <item row="0" column="0" rowspan="2">
    <widget class="ElaPlainTextEdit" name="plainTextEdit_log"/>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_port">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_0" stretch="0,1,0">
        <item>
         <widget class="ElaText" name="label_port">
          <property name="text">
           <string>serialPort</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ElaLineEdit" name="lineEdit_serial_port"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="0,1,0">
        <item>
         <widget class="ElaText" name="label_baud">
          <property name="text">
           <string>baudRate</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ElaLineEdit" name="lineEdit_baud_rate"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
        <item>
         <widget class="ElaText" name="label_databit">
          <property name="text">
           <string>dataBit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ElaLineEdit" name="lineEdit_data_bit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
        <item>
         <widget class="ElaText" name="label_parity">
          <property name="text">
           <string>parity</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ElaLineEdit" name="lineEdit_parity"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
        <item>
         <widget class="ElaText" name="label_stopbit">
          <property name="text">
           <string>stopBit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ElaLineEdit" name="lineEdit_stop_bit"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_net">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1,0">
        <item>
         <widget class="ElaText" name="label_net_port">
          <property name="text">
           <string>NetPort</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ElaLineEdit" name="lineEdit_net_port"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ElaToggleButton" name="toggleswitch_net_open" native="true"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElaToggleButton</class>
   <extends>QWidget</extends>
   <header location="global">ElaToggleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElaText</class>
   <extends>QLabel</extends>
   <header location="global">ElaText.h</header>
  </customwidget>
  <customwidget>
   <class>ElaLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ElaLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ElaPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">ElaPlainTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
